frontend/app-development/features/policyEditor/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import classes from './SettingsModal.module.css';
+import { Button, Heading } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import { CogIcon } from '@navikt/aksel-icons';
+import { Modal } from 'app-shared/components/Modal';
+
+/**
+ * Displays the settings modal. It displays the content inside the Modal component.
+ *
+ * @returns {React.ReactNode}
+ */
+export const SettingsModal = (): React.ReactNode => {
+  const { t } = useTranslation();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      {/* TODO - Move button to the correct place to open the modal from. Issue: #11047 */}
+      <Button onClick={() => setIsOpen(true)}>{t('settings_modal.open_button')}</Button>
+      <Modal
+        isOpen={isOpen}
+        onClose={() => setIsOpen(false)}
+        title={
+          <div className={classes.headingWrapper}>
+            <CogIcon fontSize='2rem' />
```

Comment:
Burde vært i CSS istedenfor? 😊

Reply:
Kan godt legge det over i CSS ja. Har bare fulgt guiden til [aksel.nav.no/ikoner](https://aksel.nav.no/ikoner). Det er det samme for min del hva vi gjør. 😄 
<img width="453" alt="Screenshot 2023-09-11 at 15 59 43" src="https://github.com/Altinn/altinn-studio/assets/133344438/245633e2-4ae8-42cf-b0d2-fcaa559580ca">

Reply:
Hm, flytte ut i en CSS klasse er sikkert fint. ☺️